docs/docs/lib/edge/cloudflare.mdx

```diff
@@ -0,0 +1,351 @@
+
+```ts
+import { handleRequest } from "@growthbook/edge-cloudflare";
+import { parse } from "cookie";
+
+export default {
+  fetch: async function (request, env, ctx) {
+    const cookie = parse(request.headers.get("Cookie") || "");
+    const config = {
+      attributes: {
+        userType: cookie["userId"] ? "logged in" : "anonymous"
+      }
+    };
+    return await handleRequest(request, env, config);
+  },
+};
+```
+
+#### More customization options
+
+For a full list of customizations, view our vendor-agnostic [Edge Utility repository <ExternalLink />](https://github.com/growthbook/growthbook-proxy/tree/main/packages/lib/edge-utils).
+
+### Set up a Payload Cache
+
+You can configure GrowthBook payload caching by using a [CloudFlare KV <ExternalLink />](https://developers.cloudflare.com/kv/reference/how-kv-works/) store. This eliminates network requests from your edge to GrowthBook which speeds up page delivery while reducing network costs.
+
+Our CloudFlare Edge App will automatically use either webhook-based or just-in-time payload caching (or both) depending on how you've set up your KV namespaces, bindings, and SDK Webhooks.
+
+More information about setting up your payload cache can be found in the [Payload Caching with CloudFlare KV Store](#payload-caching-with-cloudflare-kv-store) doc section below.
+
+### Tracking Experiment Views
+
+Running A/B tests requires a [tracking callback](/lib/js#experimentation-ab-testing). Our turnkey Edge App defaults to using built-in front-end tracking. The tracking call automatically integrates with Segment.io, GA4, and Google Tag Manager by using the mechanism outlined in our [HTML Script Tag](/lib/script-tag#tracking-experiment-views). In order to do this, the app keeps track of tracking calls triggered on edge and injects them into the front-end SDK to be automatically triggered on page load.
+
+You may wish to either customize front-end tracking or switch to edge tracking (or use both concurrently if running hybrid edge + front-end experiments).
+
+To change the front-end tracking callback, set the `GROWTHBOOK_TRACKING_CALLBACK` to your custom tracking JS code:
+
+```bash
+GROWTHBOOK_TRACKING_CALLBACK="(experiment, results) => { console.log('browser tracking callback', {experiment, results}); }"
+```
+
+To track on edge, you must inject your own tracking callback into the edge request handler code. Any experiments that run on edge will use the edge tracking callback and not the front-end callback (hybrid edge + front-end experiments being an exception):
+
+```ts
+import { handleRequest } from "@growthbook/edge-cloudflare";
+
+export default {
+  fetch: async function (request, env, ctx) {
+    const config = {
+      edgeTrackingCallback: (experiment, results) => {
+        console.log('edge tracking callback', {experiment, results});
```

Comment:
```suggestion
        console.log('edge tracking callback', {experiment, results});
        // replace console.log with your tracking library
```

IDK people somtimes make this mistake and it makes it clear you have to make a change